Better define a redeploy (#1808)
platformsh/platformsh-docs: docs/src/configuration/app/build.md

Finished performing updates.
```

> Your `deploy` hook is tied to commits in the same way as your builds. That is, once a commit has been pushed and a new build image has been created, both the `build` and `deplo`y hooks will be resused during operations that are not caused by subsequent commits. 

Reviewer: ```suggestion
Your `deploy` hook is tied to commits in the same way as your builds. That is, once a commit has been pushed and a new build image has been created, the result of both `build` and `deploy` hooks will be resused, unless there's a new git commit. 
```

> This means that adding variables, changing access permissions, or even running a `redeploy` using the CLI or management console will not cause the `deploy `hook to run again for the current commit. Redeploys trigger only the `post_deploy` hook to run again from the beginning, and a committed change to the application is needed to rerun the `build` and `deploy` hooks. 

Reviewer: ```suggestion
Redeploys trigger only the `post_deploy` hook to run again from the beginning, and a committed change to the application is needed to rerun the `build` and `deploy` hooks.
This means that adding variables, changing access permissions, or triggering a `redeploy` using the CLI or management console will not cause the `deploy `hook to run again for the current commit. 
```

The `post_deploy` hook logs to its own file in addition to the activity log, `/var/log/post-deploy.log`.

> The `post_deploy` hook is the only hook provided that will run from the beginning during a redeploy. 

Reviewer 2: @chadwcarlson @MariadeAnton 

> from the beginning

I guess you're saying it runs "every time"? I'm not sure what the beginning refers to

Reviewer 2: ... googled.. maybe that's a false friend of _d'emblée_? :fr:
